distribution/docs/src/main/resources/content/_running/platform-commands.adoc

```diff
@@ -0,0 +1,28 @@
+:title: Platform Commands
+:type: subMaintaining
+:status: published
+:parent: Available Console Commands
+:summary: Platform commands available.
+:order: 03
+
+== {title}
+
+[NOTE]
+====
+The Platform Commands are installed when the ${ddf-platform} application is installed.
```

Comment:
This seems like a strange note. You can't really use DDF at all without the platform app.

Reply:
removed